Dockerfile

```diff
@@ -34,12 +34,15 @@ RUN apt-get update && apt-get -y install software-properties-common \
         git \
     && make deps-ubuntu \
     && pip3 install --upgrade pip setuptools \
+    && hash -r \
```

Comment:
This rebuilds `$PATH`, right - why is that necessary?

Reply:
Because after the above command we actually have two competing versions installed:
- /usr/bin/pip → /usr/lib/python3/dist-packages/pip (v20)
- /usr/local/bin/pip → /usr/local/lib/python3.8/dist-packages/pip (v23)

And to prefer the newer one, you need to rehash or restart the shell – see [here](https://github.com/NVIDIA/tensorflow/issues/86#issuecomment-1490923653)